firestore.rules

```diff
@@ -1,8 +1,25 @@
 rules_version = '2';
 service cloud.firestore {
   match /databases/{database}/documents {
-    match /{document=**} {
-      allow read, write: if true;
-    }
+        match /items/{itemId} {
+          allow create: if request.auth != null;
```

Comment:
Maybe we could use the save rule as the update, delete. This was are sure that the userId is correct when creating an item

Reply:
Tried, doesn't work...